en/streaming-integrator/docs/examples/handling-requests-with-errors.md

```diff
@@ -152,9 +181,11 @@ To simulate REST API calls for the purpose, follow the procedure below:
 
     ![Imported Collections](../../images/handling-requests-with-errors/Postman.png)
     
-3. Under **Siddhi-Re-Stream Events**, select **Invalid Attribute** and click **Send**. This executes the collection to send an event with erroneous mapping. The error is indicated as follows:
+3. Under **Siddhi-Re-Stream Events**, select **Invalid Attribute** and click **Send**. 
```

Comment:
Instead of giving Postman collections, shall we give the curl command for that API call? It will be much easier for a user IMHO.

Reply:
Used this CURL command and removed all information related to Postman.